packages/teuchos/core/src/Teuchos_RCP.hpp

```diff
@@ -281,6 +281,15 @@ RCP<T>::RCP(const RCP<T>& r_ptr)
 {}
 
 
+template<class T>
+inline
+RCP<T>::RCP(RCP<T>&& r_ptr)
+  : ptr_(r_ptr.ptr_), node_(std::move(r_ptr.node_))
+{
+  r_ptr.ptr_ = 0;
```

Comment:
Correct C++11 idiom is to use `nullptr`, not `0`.

Reply:
I think tried that and it did not compile.  Is there a header that needs to be included?

Reply:
`nullptr` is a keyword, but its type is defined in `<cstddef>`.